downstream/modules/aap-hardening/ref-sensitive-variables-install-inventory.adoc

```diff
@@ -3,38 +3,33 @@
 
 [id="ref-sensitive-variables-install-inventory_{context}"]
 
-= Sensitive variables in the installation inventory
+= Securing sensitive variables with ansible vault
```

Comment:
Hey @ianf77 ! I've done a second editorial review of this section, this time correcting for style guide alignment and modular documentation alignment. I've made suggested changes in the [google doc](https://docs.google.com/document/d/1rR0tOkpPOIG1q3CrPF7PS2q5ehAioJxN2EZp002Nr-s/edit?tab=t.0) for sections 2.2.4, 2.2.4.1, and 2.2.4.2. Could you take a look at these and let me know if you're happy to transfer those changes to this PR? Thanks!